sphinx_lint.py

```diff
@@ -7,8 +7,11 @@
 
 def store_and_count_failures(clones_dir: str, repo: str, language_code: str) -> int:
     failed_checks = list(chain.from_iterable(yield_failures(clones_dir, repo)))
-    filepath = Path(f'warnings-lint-{language_code}.txt')
-    filepath.write_text('\n'.join([str(c) for c in failed_checks]))
+    log = '\n'.join([str(c) for c in failed_checks]).replace(
+        str(Path(clones_dir, 'rebased_translations', repo)) + '/', ''
+    )
```

Comment:
It would be better to give the `Path()` expression a name, though.
```suggestion
    log = '\n'.join(map(str, failed_checks)).replace(
        f'{Path(clones_dir, 'rebased_translations', repo)}'/', ''
    )
```